packages/quantic/cypress/page-objects/search.ts

```diff
@@ -6,10 +6,18 @@ import {
   // eslint-disable-next-line node/no-unpublished-import
 } from 'cypress/types/net-stubbing';
 
+type RequestParams = Record<string, string | number | boolean | undefined>;
+
 function uaAlias(eventName: string) {
   return `@UA-${eventName}`;
 }
 
+function compare(superset: RequestParams, subset: RequestParams) {
```

Comment:
I guess `includes` might be a better name for this function. Typically, `compare` refers to the equality operator.